code for tinkering audio/audio_Envelops_And_Echos.py

```diff
@@ -0,0 +1,17 @@
+#audio envelops and echos.
+
```

Comment:
Consider the use of more appropriate variable names in this function. Bonus points available if you can use an array rather than explicitly named variables (e.g. s1, s2, s3, etc.).